code/backtracking/n_queen/nqueen.py

```diff
@@ -0,0 +1,48 @@
+
+import json
```

Comment:
It seems we don't need jsom library here, could you remove this unused imprort?

Reply:
Oops. Got past me. Will remove it.